config/stage.json

```diff
@@ -139,7 +139,7 @@
 		"woocommerce/extension-promotions": true,
 		"woocommerce/extension-reviews": true,
 		"woocommerce/extension-settings": true,
-		"woocommerce/extension-settings-email": true,
+		"woocommerce/extension-settings-email": false,
```

Comment:
Same as production... and in wpcalypso.

Reply:
Great catch. I should have seen that :/ It looks like it needs to be true and that I will need a separate flag for the general email settings work! Thanks.

Reply:
@budzanowski Does that mean this PR should be closed then?

Reply:
No. I just need to make a quick change and this will be good to go.